functions-and-scope.js

```diff
@@ -72,12 +131,36 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+function highestGrade() {
+    let max = -1;
```

Comment:
Met zulke functies is het eigenlijk nooit goed om zelf de eerste waarde te verzinnen. Stel je voor dat alles cijfers onder de -1 zijn, dan klopt de uitkomst van deze functie niet. Het is daarom beter om als beginwaarde de eerste waarde van de binnenkomende array te pakken: `let max = grades[0];`